src/backends/tensorrt/tensorrtlib.cc

```diff
@@ -182,11 +182,9 @@ namespace dd
       }
 
     _builder = std::shared_ptr<nvinfer1::IBuilder>(
-        nvinfer1::createInferBuilder(trtLogger),
-        [=](nvinfer1::IBuilder *b) { b->destroy(); });
```

Comment:
These are not needed anymore?

Reply:
all destroy() methods are deprecated according to nvidia doc see for instance https://docs.nvidia.com/deeplearning/tensorrt/api/c_api/classnvinfer1_1_1_i_builder.html#a5804ba142077d478ddd77f5279b2f37a